src/main/java/network/brightspots/rcv/Utils.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Ranked Choice Voting Universal Tabulator
+ * Copyright (c) 2017-2019 Bright Spots Developers.
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU Affero General Public License as published by the Free Software Foundation, either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+ * the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with this
+ * program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package network.brightspots.rcv;
+
+class Utils {
```

Comment:
Maybe we should just merge `FileUtils` into this?

Reply:
I initially did that, but then thought it was gross to have the cached FileUtils-specific fields in the class mixing with this. Feel free to file a future issue to handle if you prefer though.